tests/unit/test_mac_frame.cpp

```diff
@@ -574,7 +574,7 @@ void TestMacFrameAckGeneration(void)
     uint8_t     ie_data[6] = {0x04, 0x0d, 0x21, 0x0c, 0x35, 0x0c};
     Mac::CslIe *csl;
 
-    ackFrame.GenerateEnhAck(receivedFrame, false, ie_data, sizeof(ie_data));
+    IgnoreError(ackFrame.GenerateEnhAck(receivedFrame, false, ie_data, sizeof(ie_data)));
```

Comment:
Why wasn't this captured by continuous integration?

Reply:
https://github.com/openthread/openthread/blob/master/include/openthread/platform/toolchain.h#L72
I think the reason is that `IgnoreError` only works for `clang`. And it seems that in the matrices there's no building using `clang` with `THREAD_VERSION=1.2` and this check is wrapped in `THREAD_VERSION=1.2`.

Reply:
Makes sense. Can we add a clang build for Thread 1.2 (can be in a separate PR)?

Reply:
Yes, just plan to do so. I'll do it.